### ISSUE TITLE
Name ordering bug

### ISSUE THREAD

**Issue by reaperhulk** [PyCQA/flake8-import-order#47]
```
from cryptography.x509.oid import (
    _OID_NAMES, OID_AUTHORITY_INFORMATION_ACCESS
    OID_AUTHORITY_KEY_IDENTIFIER
```
Gives "Imported names are in the wrong order. Should be _OID_NAMES, OID_AUTHORITY_INFORMATION_ACCESS, OID_AUTHORITY_KEY_IDENTIFIER".

Changing the import to 
```
from cryptography.x509.oid import (
    OID_AUTHORITY_INFORMATION_ACCESS
    OID_AUTHORITY_KEY_IDENTIFIER, _OID_NAMES
```
makes it happy.

There appears to be a similar bug with the output msg and its expected ordering related to capitalization as well, although I don't have an example handy right now. This is with flake8-import-order 0.6.1

**Comment by untitaker** [comment]
Regression from: https://github.com/public/flake8-import-order/pull/45

Basically that PR duplicated the sorting logic.

**Comment by pgjones** [comment]
@reaperhulk Is this fixed in 707d07ca1b53c7aa81c020fd56361b0f19303cec, #57? It looks that way to me but I may have missed your point.
